app/controllers/questions_controller.rb

```diff
@@ -27,6 +27,15 @@ def index
       .paginate(page: params[:page], per_page: 24)
   end
 
+  def index_shadow
+    @title = 'Questions and Answers'
+    set_sidebar
```

Comment:
Is this line maybe not needed?

Reply:
Yes. I'll fix it.

Reply:
Hi @jywarren Did you mean we don't need index_shadow in controller ?
Thanks !!

Reply:
But here that would be the method doing whole stuff in this PR.

Reply:
just the `set_sidebar` part, actually, sorry for ambiguity!